truss/templates/server/requirements.txt

```diff
@@ -7,7 +7,7 @@ fastapi==0.114.1
 joblib==1.2.0
 loguru==0.7.2
 msgpack-numpy==0.4.8
-msgpack==1.0.2
+msgpack==1.1.0  # Numpy/msgpack versions are finniky (1.0.2 breaks), double check when changing.
```

Comment:
what's the nature of the breakage? is this version of msgpack backwards compatible (want to make sure we're not breaking anybody here)

Reply:
https://basetenlabs.slack.com/archives/C014JUXFGLU/p1733163299697449